util/src/main/java/io/kubernetes/client/Copy.java

```diff
@@ -150,14 +150,6 @@ public void copyDirectoryFromPod(
         }
       }
     }
-    try {
-      int status = proc.waitFor();
```

Comment:
what's the point of deleting this? the method is always supposed to be hanging until the copy completes, isn't it?

Reply:
When the response from k8s API (exit call) is lightning fast, `ExecProcess.this.notifyAll();` gets called before `proc.waitFor();` and this method gets hanged for an infinite duration as no one is left to call `notify()` or `notifyAll()` method on this `proc` object.

Reply:
> m how do you think about replacing the `wait/notify` w/ `CountDownLatch`? 

`CountDownLatch` seems to be a good idea, let me try it out.